metadata.json

```diff
@@ -22,7 +22,7 @@
     },
     {
       "name": "puppetlabs/mongodb",
-      "version_requirement": ">= 0.10.0 < 1.0.0"
+      "version_requirement": ">= 0.17.0 < 1.0.0"
```

Comment:
puppetlabs/mongodb hasn't always provided this fact and may have been too broken before https://github.com/puppetlabs/puppetlabs-mongodb/commit/81ac1d7ff7267255f3de4653bd26c90feb0b829c